src/main/java/webApp/webController.java

```diff
@@ -25,32 +27,42 @@ public String index(Model model){
     @PostMapping(value = "/createSurvey", consumes = "application/json", produces = "application/json")
     @ResponseBody
     public Response createSurvey(@RequestBody SurveyMessage surveyMessage) {
-        Survey survey = new Survey(surveyMessage.getName());
+
         Collection<Question> questionList = new ArrayList<>();
 
+        ArrayList<String> questionTexts = new ArrayList<>();
+
         for (QuestionMessage question : surveyMessage.getQuestions()) {
+            questionTexts.add(question.getQuestion());
             if (question.getType().equals("openEnded")) {
                 questionList.add(new OpenEndedQuestion(question.getQuestion()));
             } else if(question.getType().equals("numberQuestion")) {
                 questionList.add(new NumberQuestion(question.getQuestion(), question.getMin(), question.getMax()));
             }
         }
 
+        HashSet<String> set = new HashSet<>(questionTexts);
```

Comment:
I am happy to see this because I wanted a HashSet to be used for this information